test/snapshots.test.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
```

Comment:
🤔

I guess you can access the mock document that's being used by enzyme. I think it was JSDOM. But it could be difficult to get it.